### PR TITLE
Add docstring to `util.resample`

### LINKED ISSUE #96: docstring: resample
child of #73

### PR DESCRIPTION
**Pull request by JensGM**
closes #96

### CODE REVIEW

File: camille/util/resample.py [hunk 1]
```diff
@@ -26,14 +26,86 @@ def rm_dupl_indices(ts):
...
+    interpolation
+
+    >>> b = camille.util.resample(a, onto=df, interp='nearest')
+    >>> b.index.equals(df.index)
+    True
+
+    Resample series `a` onto datetime index `idx`
+
+    >>> idx = pd.DatetimeIndex([
+        datetime.datetime(2018, 1, 1, 0, 0, 0, tzinfo=pytz.utc),
```
[the remark this answers is not in the document]
Comment (JensGM) [reply camille/util/resample.py]: Fixed, thank you

File: camille/util/resample.py [hunk 2]
```diff
@@ -26,14 +26,86 @@ def rm_dupl_indices(ts):
...
+    >>> idx = pd.DatetimeIndex([
+    ...     datetime.datetime(2018, 1, 1, 0, 0, 0, tzinfo=pytz.utc),
+    ...     datetime.datetime(2018, 1, 1, 0, 0, 1, tzinfo=pytz.utc),
+    ...     datetime.datetime(2018, 1, 1, 0, 0, 2, tzinfo=pytz.utc),
+    ...     datetime.datetime(2018, 1, 1, 0, 0, 3, tzinfo=pytz.utc),
+    ... ])
+    >>> b = camille.util.resample(a, onto=idx)
+    >>> b.index
+    DatetimeIndex(['2018-01-01 00:00:00+00:00', '2018-01-01 00:00:01+00:00',
+                   '2018-01-01 00:00:02+00:00', '2018-01-01 00:00:03+00:00'],
```
Comment (achaikou) [inline camille/util/resample.py]: This was 'the next example' :smile:

Comment (JensGM) [reply camille/util/resample.py]: This is the result though

Comment (achaikou) [reply camille/util/resample.py]: Oh, or course, you are right, sorry for confusion :innocent:

**Pull request merged**